firefox-ios/firefox-ios-tests/Tests/XCUITests/FindInPageTests.swift

```diff
@@ -46,31 +46,32 @@ class FindInPageTests: BaseTestCase {
         openFindInPageFromMenu(openSite: userState.url!)
 
         // Enter some text to start finding
-        app.textFields["FindInPage.searchField"].typeText("Book")
+        app.searchFields["find.searchField"].typeText("Book")
 
         // Once there are matches, test previous/next buttons
-        mozWaitForElementToExist(app.staticTexts["1/6"], timeout: TIMEOUT)
-        XCTAssertTrue(app.staticTexts["1/6"].exists)
+        mozWaitForElementToExist(app.staticTexts["1 of 6"], timeout: TIMEOUT)
+        XCTAssertTrue(app.staticTexts["1 of 6"].exists)
 
         let nextInPageResultButton = app.buttons[AccessibilityIdentifiers.FindInPage.findNextButton]
         nextInPageResultButton.tap()
-        mozWaitForElementToExist(app.staticTexts["2/6"], timeout: TIMEOUT)
-        XCTAssertTrue(app.staticTexts["2/6"].exists)
+        mozWaitForElementToExist(app.staticTexts["2 of 6"], timeout: TIMEOUT)
+        XCTAssertTrue(app.staticTexts["2 of 6"].exists)
 
-        dismissSurveyPrompt()
+        // FIXME What is the survey prompt and is it expected that we don't see it now?
+//        dismissSurveyPrompt()
```

Comment:
This is the microsurvey prompt feature that I am working on. Yes, because since the trigger has change I will need to come back and audit these tests.